packages/oui-action-menu/src/action-menu-item.controller.js

```diff
@@ -0,0 +1,54 @@
+import { addBooleanParameter } from '@oui-angular/common/component-utils'
+
+const baseClass = 'oui-action-menu__item'
+const dividerClass = 'oui-action-menu__item_divider'
+const buttonClasses = 'oui-button oui-button_action-menu'
+
+const linkTemplate = (dividerClass, external) => `
```

Comment:
If you prefer, an alternative without interpolation:
https://github.com/ovh-ux/ovh-ui-angular/pull/17